### PR TITLE
[breaking] remove addLambda()

### PR DESCRIPTION
**Pull request by nemtsov**

### CODE REVIEW

File: test/specs/index-spec.js [hunk 1]
```diff
@@ -32,15 +32,17 @@ describe('be-hogan-compiler', function() {
 
   describe('addLambda', function() {
     it('should add a lambda to the tempalte', function(done) {
```
Comment (kaicataldo) [inline test/specs/index-spec.js]: unrelated, but typo in `template`

File: test/specs/index-spec.js [hunk 2]
```diff
@@ -32,15 +32,17 @@ describe('be-hogan-compiler', function() {
 
   describe('addLambda', function() {
```
Comment (kaicataldo) [inline test/specs/index-spec.js]: Should this description change?

**Comment by agaripian** [comment]
👍

File: test/specs/index-spec.js (hunk 2 above)
Comment (nemtsov) [inline test/specs/index-spec.js]: Well, we're still adding a lambda, just without the function.

File: test/specs/index-spec.js (hunk 1 above)
Comment (nemtsov) [inline test/specs/index-spec.js]: This is my most typoed word. Period. Good catch! Fixing.

File: test/specs/index-spec.js (hunk 2 above)
Comment (kaicataldo) [inline test/specs/index-spec.js]: Right - just seems weird to leave the function name when it's being removed. Not a blocker though.

**Pull request merged**